nighres/brain/extract_brain_region.py

```diff
@@ -173,15 +179,26 @@ def extract_brain_region(segmentation, levelset_boundary,
             and os.path.isfile(bg_lvl_file) :
 
             print("skip computation (use existing results)")
-            output = {'inside_mask': load_volume(ins_mask_file),
-                      'inside_proba': load_volume(ins_proba_file),
-                      'inside_lvl': load_volume(ins_lvl_file),
-                      'region_mask': load_volume(reg_mask_file),
-                      'region_proba': load_volume(reg_proba_file),
-                      'region_lvl': load_volume(reg_lvl_file),
-                      'background_mask': load_volume(bg_mask_file),
-                      'background_proba': load_volume(bg_proba_file),
-                      'background_lvl': load_volume(bg_lvl_file)}
+            if not return_filename:
```

Comment:
Shouldn't it be the other way around? This way the object is returned when return_filename is True

Reply:
Hi @juhuntenburg, it was my mistake. I fixed it and pushed a new commit. Please check for it.